lib/scala/syntax/definition/src/main/scala/org/enso/syntax/text/ast/Doc.scala

```diff
@@ -543,6 +543,10 @@ object Doc {
         case object Removed    extends Type
         case object Modified   extends Type
         case object Upcoming   extends Type
+        case object Private    extends Type
+        case object Advanced   extends Type
+        case object TextOnly   extends Type
+        case object Unstable   extends Type
```

Comment:
Please sort the list of these in the docs alphabetically, and do so here as well.